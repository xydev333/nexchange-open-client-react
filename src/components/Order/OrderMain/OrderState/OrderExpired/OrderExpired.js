import React, { Component } from 'react';
import styles from '../OrderState.scss';

class OrderExpired extends Component {
  componentDidMount() {
    $('[data-toggle="tooltip"], [rel="tooltip"]').tooltip();
  }

  render() {
    return (
      <div className={styles.container}>
        <h2 className={styles.title}>Order Expired</h2>
        <h3 className={styles.subtitle}>Your order is expired. You can make a new exchange order.</h3>

        <h4
          className={styles.warning}
          data-toggle="tooltip"
          data-placement="top"
          data-original-title="
						If you have already sent your coins, donâ€™t worry.
						Once we get them, the status of the order will be updated retroactively."
        >
          Already sent?
        </h4>
      </div>
    );
  }
}

export default OrderExpired;
