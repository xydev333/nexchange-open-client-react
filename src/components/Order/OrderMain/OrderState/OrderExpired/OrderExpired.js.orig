import React, { Component } from 'react';
import OrderAlert from '../OrderIcons/OrderAlert/OrderAlert';
<<<<<<< HEAD
import { I18n, Trans } from 'react-i18next';
=======
>>>>>>> Improve order expired state.
import styles from '../OrderState.scss';

class OrderExpired extends Component {
  componentDidMount() {
    $('[data-toggle="tooltip"], [rel="tooltip"]').tooltip();
  }

  render() {
    return (
     <I18n ns="translations">
     {(t) => (        
      <div className={styles.container}>
        <OrderAlert />
<<<<<<< HEAD
        <h2 className={styles.title}>{t('order.expired1')}</h2>
        
        <Trans i18nKey="order.expired2">
=======
        <h2 className={styles.title}>Order Expired</h2>
>>>>>>> Improve order expired state.
        <h3 className={styles.subtitle}>
          Your order is expired. You can make a{' '}
          <a href="/" className={styles['new-order-link']}>
            new exchange order
          </a>.
        </h3>
<<<<<<< HEAD
        </Trans>
=======
>>>>>>> Improve order expired state.

        <h4
          className={styles.warning}
          data-toggle="tooltip"
          data-placement="top"
          data-original-title={t('order.expired3')}
        >
          {t('order.expired4')}
        </h4>
      </div>
	  )}
	 </I18n>
    );
  }
}

export default OrderExpired;
