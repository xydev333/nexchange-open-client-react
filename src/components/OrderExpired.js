import React, { Component } from 'react';
import {Icon} from 'react-fa';


class OrderExpired extends Component {
	constructor(props) {
		super(props);
	}

	componentDidMount() {
		$('[data-toggle="tooltip"], [rel="tooltip"]').tooltip();
	}

	render() {
	    return (
			<div className="text-center">
			    <h2 style={{margin: "0"}}>Order Expired</h2>
			    <h5>Your order is expired. You can make a new exchange order.</h5>

			    <h4 style={{margin: "25px 0 5px", "fontWeight": "500"}} className="text-warning" data-toggle="tooltip" data-placement="top" title="" data-original-title="If you just sent, don't worry. Once we receive your coins, status of your order will be updated.">Already sent?</h4>
			</div>
	    );
	}
}

export default OrderExpired;
