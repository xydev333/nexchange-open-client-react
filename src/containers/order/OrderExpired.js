import React, { Component } from 'react';

class OrderExpired extends Component {
	componentDidMount() {
		$('[data-toggle="tooltip"], [rel="tooltip"]').tooltip();
	}

	render() {
	    return (
			<div className="text-center">
				<h2>Order Expired</h2>
				<h5>Your order is expired. You can make a new exchange order.</h5>
				
				<div>
					<h4
						className="text-warning"
						data-toggle="tooltip"
						data-placement="top"
						data-original-title="If you have already sent your coins, donâ€™t worry. Once we get them, the status of the order will be updated retroactively.">
						Already sent?
					</h4>
				</div>
			</div>
	    );
	}
}

export default OrderExpired;
