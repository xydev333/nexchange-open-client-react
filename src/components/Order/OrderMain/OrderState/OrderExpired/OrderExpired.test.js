import React from 'react';
import { shallow } from 'enzyme';
import OrderExpired from './OrderExpired';
window.$ = window.jQuery = require('jquery');
require('Js/bootstrap.min.js');

describe('OrderExpired', () => {
  it('renders correctly', () => {
    expect(shallow(<OrderExpired />)).toMatchSnapshot();
  });
});
