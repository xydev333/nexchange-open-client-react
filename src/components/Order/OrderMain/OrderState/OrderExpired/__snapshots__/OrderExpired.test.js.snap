// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderExpired renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <OrderExpired />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <OrderAlert />,
        <h2
          className="title"
>
          Order Expired
</h2>,
        <h3
          className="subtitle"
>
          Your order is expired. You can make a
           
          <a
                    className="new-order-link"
                    href="/"
          >
                    new exchange order
          </a>
          .
</h3>,
        <h4
          className="warning"
          data-original-title=" If you have already sent your coins, don’t worry. Once we get them, the status of the order will be updated retroactively."
          data-placement="top"
          data-toggle="tooltip"
>
          Already sent?
</h4>,
      ],
      "className": "container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Order Expired",
          "className": "title",
        },
        "ref": null,
        "rendered": "Order Expired",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Your order is expired. You can make a",
            " ",
            <a
              className="new-order-link"
              href="/"
>
              new exchange order
</a>,
            ".",
          ],
          "className": "subtitle",
        },
        "ref": null,
        "rendered": Array [
          "Your order is expired. You can make a",
          " ",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "new exchange order",
              "className": "new-order-link",
              "href": "/",
            },
            "ref": null,
            "rendered": "new exchange order",
            "type": "a",
          },
          ".",
        ],
        "type": "h3",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Already sent?",
          "className": "warning",
          "data-original-title": " If you have already sent your coins, don’t worry. Once we get them, the status of the order will be updated retroactively.",
          "data-placement": "top",
          "data-toggle": "tooltip",
        },
        "ref": null,
        "rendered": "Already sent?",
        "type": "h4",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <OrderAlert />,
          <h2
            className="title"
>
            Order Expired
</h2>,
          <h3
            className="subtitle"
>
            Your order is expired. You can make a
             
            <a
                        className="new-order-link"
                        href="/"
            >
                        new exchange order
            </a>
            .
</h3>,
          <h4
            className="warning"
            data-original-title=" If you have already sent your coins, don’t worry. Once we get them, the status of the order will be updated retroactively."
            data-placement="top"
            data-toggle="tooltip"
>
            Already sent?
</h4>,
        ],
        "className": "container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Order Expired",
            "className": "title",
          },
          "ref": null,
          "rendered": "Order Expired",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Your order is expired. You can make a",
              " ",
              <a
                className="new-order-link"
                href="/"
>
                new exchange order
</a>,
              ".",
            ],
            "className": "subtitle",
          },
          "ref": null,
          "rendered": Array [
            "Your order is expired. You can make a",
            " ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "new exchange order",
                "className": "new-order-link",
                "href": "/",
              },
              "ref": null,
              "rendered": "new exchange order",
              "type": "a",
            },
            ".",
          ],
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Already sent?",
            "className": "warning",
            "data-original-title": " If you have already sent your coins, don’t worry. Once we get them, the status of the order will be updated retroactively.",
            "data-placement": "top",
            "data-toggle": "tooltip",
          },
          "ref": null,
          "rendered": "Already sent?",
          "type": "h4",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
